### ISSUE TITLE
Support multi namespace in config

### ISSUE THREAD

**Issue by arezvani** [k8spin/prometheus-multi-tenant-proxy#45]
Hi,
How can I have multi namespace for each user in config?

**Comment by angelbarrera92** [comment]
Currently, there's no way... but we are open to new ideas and suggestions :)
How do you imagine this feature?

**Comment by arezvani** [comment]
Thank you
We have k8s cluster and each user have many namespace. I want to give each user, 1 endpoint and 1 user that can see just metrics of his namespaces.

**Comment by angelbarrera92** [comment]
> @angelbarrera92 Thank you We have k8s cluster and each user have many namespace. I want to give each user, 1 endpoint and 1 user that can see just metrics of his namespaces.

Hi!

Let me know if this PR makes sense to you: https://github.com/k8spin/prometheus-multi-tenant-proxy/pull/46/files

TL;DR it enables you to configure the proxy with:

```yaml
users:
  - username: Happy
    password: Prometheus
    namespace: default
  - username: Sad
    password: Prometheus
    namespace: kube-system
  - username: Multiple
    password: Namespaces
    namespace: monitoring
    namespaces:
      - default
      - kube-system
      - kube-system
```

**Comment by arezvani** [comment]
Thank you so much
